CREDITS.md

```diff
@@ -10,151 +10,152 @@
 
 ### TileSets
 
-- FantasyHex Merchant of Venice by Herobrine
 - Absolute Units by Basil
 - Brazilwood Camp by Kurwizimi
+- FantasyHex Merchant of Venice by Herobrine
+- HexaRealm Chateau by Firelord
 - Various units by Dom Pedro III
-- HexaRalm Chateau by Firelord
 
 ### Icons
 
-- Worldwide by iconsdb.com for Exploration policy branch
-- Feather by iconsdb.com for Aesthetics policy branch
-- Headdress by Arturo Alejandro Romo Escartin from NounProject.com
-- Track by andriwidodo from NounProject.com
-- Tomahawk by Bakunetsu Kaito from NounProject.com
-- Diamond by SHAHAREA from NounProject.com
-- Ship by Robert Bjurshagen from NounProject.com
+- Anchor by Alvaro Cabrera from NounProject.com
+- Art by scribble.glyph from NounProject.com
+- Artist by my name is mud from NounProject.com
+- Axe by Vectors Market from NounProject.com
+- Bazooka by Visualeat from NounProject.com
+- Beaker by ibrandify from NounProject.com
+- Building by Taylan Sentürk from NounProject.com
+- Capitalism by Adrien Coquet from NounProject.com
 - Caravel by Anthony Bossard from NounProject.com
-- Tower by Andrejs Kirma from NounProject.com
-- [Pracinha](https://thenounproject.com/icon/snake-7803482/) by Md Ahasan Habib
-- Paint by tezar tantular from NounProject.com
-- Fort by Anton from NounProject.com
-- Turban by Andrejs Kirma from NounProject.com
-- Taurus by Erika Jasso from NounProject.com
-- Library by Rob Crosswell from NounProject.com
-- Shield Zulu by Smashing Stocks from NounProject.com
-- Kris by Mochammad Kafi from NounProject.com
-- Wing by prasong tadoungsorn from NounProject.com
-- Horse Stable by Azam Ishaq from NounProject.com
-- Shield Zulu by Smashing Stocks from NounProject.com
 - Chateau Castle by Vectors Market from NounProject.com
-- Pentagram by Adame Dahmani from NounProject.com
-- Bazooka by Visualeat from NounProject.com
-- Jetpack by HideMaru from NounProject.com
-- Axe by Vectors Market from NounProject.com
-- Christianity Cross by ProSymbols from NounProject.com
+- Chocolate by Vector Portal from NounProject.com
 - Christianity by Ben Davis from NounProject.com
 - Christianity by Samsul Rizal from NounProject.com
-- Winged Lion by Studio Fibonacci from NounProject.com
+- Christianity Cross by ProSymbols from NounProject.com
```

Comment:
Could we link to the nounproject page too? It makes it easier to find the images.